e2e/playwright/support/user.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// preferences any new plain user should have to avoid tours appearing
+export const preferencesForUser = (userId: string) => {
```

Comment:
This can be more descriptive

```suggestion
export const userPreferencesForAvoidingTours = (userId: string) => {
```

This file should probably be named something like `default-user-preferences.ts`